src/machine/i2c.go

```diff
@@ -1,5 +1,4 @@
-//go:build atmega || nrf || sam || stm32 || fe310 || k210 || rp2040
-// +build atmega nrf sam stm32 fe310 k210 rp2040
+// +build atmega nrf sam stm32 fe310 k210 rp2040 esp32c3
```

Comment:
Please do not delete definitions from go 1.17 or later.

```suggestion
//go:build atmega || nrf || sam || stm32 || fe310 || k210 || rp2040 || esp32c3
// +build atmega nrf sam stm32 fe310 k210 rp2040 esp32c3
```

Reply:
my bad. I have fix it.